doc/EnvironmentVariables.md

```diff
@@ -84,7 +84,10 @@
-| MqttBroker__MaxInflightMessages         | Maximum number of QoS 1 messages that can be in the process of being sent simultaneously for a single session                                      | int32                                                                 | 16                         |
-| MqttBroker__MaxQueuedMessages           | Maximum number of messages that can be in the outgoing queue for a single session                                                                  | int32                                                                 | 1000                         |
-| MqttBroker__MaxQueuedBytes              | Maximum number of bytes that can be in the outgoing queue for a single session                                                                     | int32 (arm), int64 (amd64)                                            | 0 (no limit)                         |
-| MqttBroker__WhenFull                    | Defines what to do in case either max queued messages or max queued bytes is reached                                                               | drop_new, drop_old                                                    | drop_new                       |
\ No newline at end of file
+| MqttBroker__MaxInflightMessages         | Maximum number of QoS 1 messages that can be in the process of being sent simultaneously for a single session                                      | int32                                                                 | 16                            |
+| MqttBroker__MaxQueuedMessages           | Maximum number of messages that can be in the outgoing queue for a single session                                                                  | int32                                                                 | 1000                          |
+| MqttBroker__MaxQueuedBytes              | Maximum number of bytes that can be in the outgoing queue for a single session                                                                     | int32 (arm), int64 (amd64)                                            | 0 (no limit)                  |
+| MqttBroker__WhenFull                    | Defines what to do in case either max queued messages or max queued bytes is reached                                                               | drop_new, drop_old                                                    | drop_new                      |
+| MqttBridge__StorageMaxFileSize          | If `UsePersistentStorage` is set, defines maximum size (in bytes) of the MQTT Bridge storage. This storage is separate from edgehub messages / twins storage.   | int64                                                    | 33554432 (32mb)               |
```

Comment:
We need to think through how this will work when we have multiple files for different priority queues. 
So for now, I suggest we rename this to StorageMaxDiskSize or StorageMaxDiskUsage. 
Or alternatively we can remove it altogether till we figure out how to support it for multiple priorities (in case we need to implement it in a different way).